sites/main-site/src/content/blog/2024/tools-3_1_0.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: 'nf-core/tools - 3.1.0'
+subtitle: 'Everything is going crate'
+headerImage: 'https://images.unsplash.com/photo-1655874835023-02d7371fbe90'
+headerImageAlt: 'A wall of red, blue and green plastic crates'
+pubDate: 2024-12-10T14:00:00+01:00
+authors:
+    - 'mashehu'
+    - 'mirpedrol'
+    - 'ewels'
+label:
+    - 'tools'
+maxHeadingDepth: 3
+---
+
+This release comes with some new features and smaller pipeline template, which shouldn't cause too much of a headache.
+As always, if you have any problems or run into any bugs, reach out on the [#tools slack channel](https://nfcore.slack.com/archives/CE5LG7WMB).
+
+# Highlights
+
+## Research Object (RO) Crates in nf-core pipelines
+
+    RO Crate is an open-source standard which we use to describe our pipelines and their components in a structured way and helps with automated provenance tracking.
+    In our case, it describes all files that are in the pipeline template, as well as the pipeline authors, besides some general metadata.
+    More information about RO Crates can be found [here](https://www.researchobject.org/ro-crate/).
+    Thanks to [Phil Ewels](https://github.com/ewels), [Stian Soiland-Reyes](https://github.com/stain) and [Simone Leo ](https://github.com/simleo) for kicking off this effort and providing useful feedback!
+
+    :::note
+    This RO Crate describes the pipeline as a whole, not pipeline runs.
+    For that kind of provenance, use the new [nf-prov plugin](https://github.com/nextflow-io/nf-prov), which is currently in development.
+    :::
+
+    If you want to update the RO Crate for your pipeline, run `nf-core pipelines rocrate` (automatically run with `nf-core pipelines sync`).
+
+    For more information about the command and how it generates the RO Crate, see [the docs](https://nf-co.re/docs/nf-core-tools/pipelines/rocrate).
+
+## ORAS container URI support in `nf-core pipelines download`
+
+    Previously, you needed to use a `https://` prefix, but thanks to the work by [@MatthiasZepper](https://github.com/MatthiasZepper") we can now use the `oras://` prefix.
+    This requires singularity to be installed on your system.
+
+## `main` as optional default branch
+
+    With this release we extended the template and tooling to support either `main` or `master` as the default release branch for pipelines.
+
+If you want to use the `main` branch in your pipeline, make sure to set `defaultBranch = main` on the `manifest` section from the `nextflow.config` file. Afterwards, you can change the GitHub repository default branch to `main`.
```

Comment:
```suggestion
If you want to use the `main` branch in your pipeline, make sure to set `defaultBranch = main` on the `manifest` section from the `nextflow.config` file. Afterwards, you can change the GitHub repository default branch to `main`.

Then, you can run `nf-core pipelines sync` again. This will automatically change all the required links from `master` to `main`.
```